core/bubble.js

```diff
@@ -274,48 +274,53 @@ Blockly.Bubble.prototype.createDom_ = function(content, hasResize) {
   return this.bubbleGroup_;
 };
 
+/**
```

Comment:
Some bubble functions had to change to make block comments and workspace comments both drag with the bubble dragger.

You can play with block comments by changing the injection struct in `vertical_playground.html` to have `comments: true`.